libduel.cpp

```diff
@@ -2189,6 +2189,24 @@ int32 scriptlib::duel_get_attack_target(lua_State *L) {
 	interpreter::card2value(L, pcard);
 	return 1;
 }
+int32 scriptlib::duel_get_battle_monster(lua_State* L) {
```

Comment:
format consistency
```suggestion
int32 scriptlib::duel_get_battle_monster(lua_State *L) {
```
also in header file below